strings.go

```diff
@@ -0,0 +1,24 @@
+package gana
+
+import "strings"
+
+// DropString allocates a new string, with the first `num` bytes of a string dropped.
+func DropString(num uint, what string) string {
+	l := uint(len(what))
```

Comment:
Do we have to make a static variable `l` here or can we just put the `uint(let(what))` in the if statement? It would be nice to see if the generated assembly is any different.

Reply:
Apologies, this was left over from before. Will push a commit that inlines it into the if statement. [Here's the assembly of both](https://gist.github.com/matthewsanetra/1529262c8bfd8b9e24ed224d1edfa912)